cnv/cnv_virtual_machines/cnv-controlling-vm-states.adoc

```diff
@@ -1,21 +1,21 @@
 [id="cnv-controlling-vm-states"]
-= Controlling virtual machines states
+= Controlling virtual machine states
 include::modules/cnv-document-attributes.adoc[]
 :context: cnv-controlling-vm-states
 toc::[]
 
-With {CNVProductName}, you can stop, start, and restart virtual machines from
-both the web console and the command-line interface (CLI).
+You can stop, start, restart, and unpause virtual machines from the web console.
```

Comment:
Hey @ousleyp, I didn't check the actual web console, but this sounds like you can `unpause` only, but not `pause`? That wouldn't make much sense to me :thinking:

Reply:
Hey @slintes, that's my understanding of the available functionality at this point. @jelkosz can you please confirm? (note that this is still a draft, I am thinking of updating it so that it describes how to unpause from both screens in order to match the other modules in this assembly)

Reply:
@ousleyp UI part lgtm